### PR TITLE
Memeolist show additional information of the meme

### PR DESCRIPTION
**Pull request by wojta**
https://issues.jboss.org/browse/AEROGEAR-7828

Shows additional information of the meme.

* Number of likes
* Author(owner)'s name

### CODE REVIEW

**Comment by wtrocki** [comment]
@wojta ping me on UI stuff :) Going to verify this :)

File: Memeolist/Memeolist/view/list-meme/MemesTableViewCell.swift [hunk 1]
```diff
@@ -8,12 +8,15 @@ class MemesTableViewCell: UITableViewCell {
...
+    @IBOutlet var user:UILabel!
+    @IBOutlet var avatar:UIImageView!
 
     func configure(with meme: AllMemesQuery.Data.AllMeme) {
         memeId = meme.id
         let url = URL(string: meme.photourl)
         memeImageView.kf.setImage(with: url, placeholder: UIImage(named: "loading"))
-        likesLabel?.text = "0 likes" //"\(meme.votes ?? 0) likes"
+        likesLabel?.text = "\(meme.likes) likes"
+        user?.text = meme.owner ?? "David Duchovny"
```
Comment (wtrocki) [inline Memeolist/Memeolist/view/list-meme/MemesTableViewCell.swift]: owner is required and it should be always set. It is not?

Comment (wojta) [reply Memeolist/Memeolist/view/list-meme/MemesTableViewCell.swift]: You're right, it is.

Comment (wtrocki) [reply Memeolist/Memeolist/view/list-meme/MemesTableViewCell.swift]: Let's remove

**Review by wtrocki — APPROVED** [review]

**Pull request merged**